spark/src/jmh/java/org/apache/iceberg/spark/data/parquet/SparkParquetReadersFlatDataBenchmark.java

```diff
@@ -116,7 +117,7 @@ public void tearDownBenchmark() {
   public void readUsingIcebergReader(Blackhole blackHole) throws IOException {
     try (CloseableIterable<InternalRow> rows = Parquet.read(Files.localInput(dataFile))
         .project(SCHEMA)
-        .createReaderFunc(type -> SparkParquetReaders.buildReader(SCHEMA, type))
+        .createReaderFunc(type -> SparkParquetReaders.buildReader(SCHEMA, type, Collections.emptyMap()))
```

Comment:
Can we add a `buildReader` method that defaults the map to empty? Then we wouldn't need to change this file at all.

Reply:
👍